### PR TITLE
NEWREADME

### PR DESCRIPTION
**Pull request by Iam2smart**
New readme not a boring one. Thanks

### CODE REVIEW

File: NEWREADME.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Supports ios
 Devices that it supports are 
 Iphone 7/7 plus 10.1.1 only
 Iphone 6s/6s plus 10.2
+Iphone SE 10.2
 Iphone 5s 10.2 BEWARE STILL A WORK IN PRGRESS ON THIS ONE BECAUSE OF THE 4K CHIP
```
Comment (cpreston321) [inline NEWREADME.md]: *PROGRESS

Comment (Iam2smart) [inline NEWREADME.md]: Which one

Comment (cpreston321) [inline NEWREADME.md]: "Iphone 5s 10.2 BEWARE STILL A WORK IN PRGRESS ON THIS ONE BECAUSE OF THE 4K CHIP"

Comment (Iam2smart) [inline NEWREADME.md]: They do have support for iphone 5s but some work and some don't so beware

Comment (cpreston321) [inline NEWREADME.md]: haha I'm just saying you spelled progress wrong it's not prgress

Comment (Iam2smart) [inline NEWREADME.md]: oh oops sorry will fix that

**Comment by Iam2smart** [comment]
Fixed the typo in the UPDATED Newreadme.md file

**Comment by Iam2smart** [comment]
Added ipad air 2 to the support list

File: NEWREADME.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+10.1.1 Iphone 7/7 plus only
+10.2 for the rest of the 64-Bit devices
+
+Devices that it supports are 
+Iphone 7/7 plus 10.1.1 only
+Iphone 6s/6s plus 10.2
+Iphone SE 10.2
+Iphone 5s 10.2 BEWARE STILL A WORK IN PROGRESS ON THIS ONE BECAUSE OF THE 4K CHIP
+Ipad PRO 10.2
+Ipad Air 2 10.2 ALSO BEWARE LIKE THE 5S WORK IN PROGRSS
```
Comment (cpreston321) [inline NEWREADME.md]: Progress

Comment (cpreston321) [inline NEWREADME.md]: haha typing quick I see

**Comment by kpwn** [comment]
Settled for another readme. Thanks anyway!

**Pull request closed without merging**